modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaBaseModelListenerCheck.java

```diff
@@ -32,41 +35,79 @@ protected String format(
 
 		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
 
-		List<String> implementedClassNames =
-			javaClass.getImplementedClassNames();
+		List<String> extendedClassNames = javaClass.getExtendedClassNames();
 
-		if (implementedClassNames.contains("BaseModelListener")) {
+		if (!extendedClassNames.contains("BaseModelListener")) {
 			return content;
 		}
 
+		_extendedClassType = _getExtendedClassType(content);
+
 		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
 			if (!childJavaTerm.isJavaMethod()) {
 				continue;
 			}
 
 			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
 
-			String newJavaMethodContent = _formatMethodDefinition(javaMethod);
+			String newJavaContent = _formatMethod(javaMethod);
 
 			content = StringUtil.replace(
-				content, javaMethod.getContent(), newJavaMethodContent);
+				content, javaMethod.getContent(), newJavaContent);
 		}
 
 		return content;
 	}
 
-	private String _formatMethodDefinition(JavaMethod javaMethod) {
-		String javaMethodContent = javaMethod.getContent();
-
+	private String _formatMethod(JavaMethod javaMethod) {
 		String javaMethodName = javaMethod.getName();
 
-		if (!javaMethodName.equals("onAfterUpdate") &&
-			!javaMethodName.equals("onBeforeUpdate")) {
+		if (javaMethodName.equals("onAfterUpdate") ||
+			javaMethodName.equals("onBeforeUpdate")) {
+
+			return _formatMethodDefinitionWithParameterUpgrade(javaMethod);
+		}
+
+		return _formatMethodDefinitionWithoutParameterUpgrade(javaMethod);
+	}
 
+	private String _formatMethodDefinitionWithoutParameterUpgrade(
+		JavaMethod javaMethod) {
+
+		String javaMethodContent = javaMethod.getContent();
+
+		Matcher matcher = _superMethodPattern.matcher(javaMethodContent);
+
+		if (!matcher.find()) {
 			return javaMethodContent;
 		}
 
-		javaMethodContent = _formatSuper(javaMethodContent);
+		String methodCall = JavaSourceUtil.getMethodCall(
+			javaMethodContent, matcher.start());
+
+		methodCall = methodCall.trim();
```

Comment:
Have to do this weird trick to clean up the match while still storing the remembered white space into matcher.group(1) maybe theres a better way to do this but this worked for me